webapp/src/main/java/org/opentestsystem/rdw/admin/web/GroupController.java

```diff
@@ -27,9 +31,14 @@
         return service.getByQuery(user, query);
     }
 
+    @PutMapping("/api/groups")
```

Comment:
The idea here is that at some point we would like the UI to be able to see deleted groups and have the ability to "undelete" them, so the soft deletion implementation would have to leak all the way to the front end, at which point this would just become another update.  This is why I'm doing this as an update.  This also sets us up for when we do edit groups.